stdlib/LinearAlgebra/test/triangular.jl

```diff
@@ -357,11 +374,16 @@ for elty1 in (Float32, Float64, BigFloat, ComplexF32, ComplexF64, Complex{BigFlo
             @test transpose(A1)*B ≈ transpose(Matrix(A1))*B
             @test A1'B ≈ Matrix(A1)'B
             @test A1*transpose(B) ≈ Matrix(A1)*transpose(B)
+            @test A1*Transpose(B) ≈ Matrix(A1)*transpose(B)
```

Comment:
How is this one different from the one above?

Reply:
I guess it's not. IIUC then `B` is a dense matrix without extra structure, and `transpose(B)` calls the wrapper `Transpose(B)`.